R/oc_forward.R

```diff
@@ -412,29 +416,35 @@ oc_forward_df <-
             )
         )
 
-      results <- tidyr::unnest(results_nest, op) # nolint
-      # `op` is necessary, so that other list columns are not unnested
-      # but lintr complains about `op` not being defined
+      if (utils::packageVersion("tidyr") > "0.8.99") {
+        results <- tidyr::unnest(results_nest, op, names_repair = "unique") # nolint
+        # specifying `op` is necessary, so that other list columns are not
+        # unnested, but lintr complains about `op` not being defined
```

Comment:
I had similar issues with another package yesterday, and can't remember whether one can use the rlang .data adverb in such cases too? But fine to leave the lintr exception, it's easier... if R CMD check doesn't complain either.

Reply:
This does work with rlang::.data (now), so I made the necessary changes.